.gitignore

```diff
@@ -11,3 +11,6 @@
 /target/
 Cargo.lock
 /testing/target
+
+# Editor-Specific Files
+.devcontainer
```

Comment:
I'm happy to remove this if it's a problem, this was solely added as a convenience to myself.